### ISSUE TITLE
Tabs not shown

### ISSUE THREAD

**Issue by loodachina** [robertstaddon/buddypress-woocommerce#3]
Plugin install went ok, Account tab appeared in buddypress profile menus but when you click on any tab from my account menu it doesn't reload any specific tab

`[01-Dec-2016 02:04:00 UTC] TEST:'/orders/'|'orders'|''|false
[01-Dec-2016 02:04:00 UTC] TEST:'/edit-address/'|'edit-address'|''|false
[01-Dec-2016 02:04:00 UTC] PHP Warning:  Missing argument 1 for BP_WooCommerce::customer_edit_account_url(), called in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 219 and defined in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 47
[01-Dec-2016 02:04:03 UTC] TEST:'/orders/'|'orders'|''|false
[01-Dec-2016 02:04:03 UTC] TEST:'/edit-address/'|'edit-address'|''|false
[01-Dec-2016 02:04:03 UTC] PHP Warning:  Missing argument 1 for BP_WooCommerce::customer_edit_account_url(), called in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 219 and defined in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 47
[01-Dec-2016 02:04:07 UTC] TEST:'/orders/'|'orders'|''|false
[01-Dec-2016 02:04:07 UTC] TEST:'/edit-address/'|'edit-address'|''|false
[01-Dec-2016 02:04:07 UTC] PHP Warning:  Missing argument 1 for BP_WooCommerce::customer_edit_account_url(), called in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 219 and defined in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 47
[01-Dec-2016 02:08:44 UTC] TEST:'/orders/'|'orders'|''|false
[01-Dec-2016 02:08:44 UTC] TEST:'/edit-address/'|'edit-address'|''|false
[01-Dec-2016 02:08:44 UTC] PHP Warning:  Missing argument 1 for BP_WooCommerce::customer_edit_account_url(), called in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 219 and defined in ../wp-content/plugins/buddypress-woocommerce/buddypress-woocommerce.php on line 47
`

**Comment by robertstaddon** [comment]
@igorlbg , I just committed https://github.com/robertstaddon/buddypress-woocommerce/commit/30c042c3a57b281181f272ea25ce64fee271a413, which I think should resolve your PHP warnings.

**Comment by loodachina** [comment]
@robertstaddon tnx for the fix, that fixed spam in the error_log but I wasn't able to make it work on one of the websites I am working on.

I was able to track it down, website had The Seo Framework plugin installed instead Yoast SEO.
Even after deactivating and completely uninstalling that plugin, account tabs were not working at all.

I did full reset, installed only woocommerce, buddypress, your plugin and it worked, as soon as I installed TSF, it went nuts, and again upon deleting it, redirects for tabs were broken.

After another reset, and redoing all plugins again without The SEO Framework, everything works as intended. Something is messing up your redirects when website have/had TSF installed so you might look for a solution for future bugs :-)

Cheers
